selfdrive/car/gm/values.py

```diff
@@ -66,16 +68,16 @@ class Footnote(Enum):
 @dataclass
 class GMCarInfo(CarInfo):
   package: str = "Adaptive Cruise"
-  harness: Enum = Harness.none
+  harness: Enum = Harness.obd_ii
 
 
 CAR_INFO: Dict[str, Union[GMCarInfo, List[GMCarInfo]]] = {
-  CAR.HOLDEN_ASTRA: GMCarInfo("Holden Astra 2017", harness=Harness.custom),
```

Comment:
@vanillagorillaa do we want to start showing these cars in the harness list? It may appear to end users all these cars need is an OBD II harness and all is well. Before they would have needed to read the footnote on the supported cars page first to know to get a custom harness.

Reply:
I believe the custom harness is represented as the developer harness, and the developer harness wouod be useless - they are all setup the same as the Volt.

The l&p harness isn't mandatory either, it's just convenient. The same basic instructions at https://zoneos.com/volt/ apply to all ASCM-bearing vehicles

Reply:
I think we still want to force people to read the footnote which lets users know they won't have a good time unless they get the ASCM harness (due to faults I believe). Until we are able to pop up with limitations when selecting a car harness, at least